packages/teuchos/kokkoscompat/src/KokkosCompat_ClassicNodeAPI_Wrapper.hpp

```diff
@@ -43,21 +44,28 @@ class KokkosDeviceWrapperNode {
   /// release.  This Node type is safe to use.
   static constexpr bool classic = false;
 
+#if defined(TPETRA_ENABLE_DEPRECATED_CODE) || !defined(TEUCHOS_HIDE_DEPRECATED_CODE)
```

Comment:
We shouldn't actually use the Tpetra macro here, since Teuchos does not depend on Tpetra.  However, the macro is likely harmless here.

Reply:
I agree, but if someone has TEUCHOS_HIDE_DEPRECATED_CODE=ON and TPETRA_ENABLE_DEPRECATED_CODE=ON, he'll get a bunch of compiler errors unless we test for TPETRA_ENABLE_DEPRECATED_CODE here.

Reply:
should be ok then